cordapp-cpk2/src/main/java/net/corda/plugins/cpb2/CpbTask.java

```diff
@@ -61,18 +65,29 @@ public CpbTask() {
     @NotNull
     public AbstractCopyTask from(@NotNull Object... args) {
         return super.from(args, copySpec ->
-            copySpec.exclude(this::isCPK)
```

Comment:
(this is how it used to be so nothing to do with the PR) 

Not a Gradle expert but I think it looks to me a bit weird to exclude when `isCPK`.